ruby.js

```diff
@@ -157,6 +200,14 @@ function activate(context) {
 		balancePairs(vscode.window.activeTextEditor.document);
 	}
 
+	which("rct-complete", function(err, found) {
```

Comment:
Thanks for your review. I don't like handling completion event even though rct-cmplete is not installed. How about to try to run `rct-complete --help` at once to check rct-complete is installed?